gradle/versions.gradle

```diff
@@ -11,7 +11,7 @@ ext.libVersions = [
     jackson: '2.5.1',
     jackson_annotation: '2.5.0',
     cassandra: '2.2.1',
-    cassandra_driver_core: '2.2.0-rc3',
+    cassandra_driver_core: '2.1.5',
```

Comment:
re: concerns

We actually have a reliable repro with the other version of the driver vs. this old one and when it's not day-before-release day we can actually find and fix the bug and help upstream, since this is an open / active / unassigned issue there.